apps/api/src/friendships/friendships.module.ts

```diff
@@ -4,11 +4,12 @@ import { FriendshipsService } from './friendships.service';
 import { Friendships as FriendshipsEntity } from 'types';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { UsersModule } from '../users/users.module';
+import { TransformUserService } from 'src/TransformUser/TransformUser.service';
 
 @Module({
   imports: [TypeOrmModule.forFeature([FriendshipsEntity]), UsersModule],
   controllers: [FriendshipsController],
-  providers: [FriendshipsService],
+  providers: [FriendshipsService, TransformUserService],
```

Comment:
If TransformUserService is in another module, export the service in TransformUserService and simply import the Module here. It should not be in the list of provider if it does not come from the same module or you run the chance of having multiple instanciations of the same service and a few other issues

Reply:
done